binderhub/tests/conftest.py

```diff
@@ -86,13 +86,6 @@ def load_mock_responses(host):
     MockAsyncHTTPClient.mocks.update(records)
 
 
-def pytest_collection_modifyitems(items):
-    """add asyncio marker to all async tests"""
-    for item in items:
-        if inspect.iscoroutinefunction(item.obj):
-            item.add_marker('asyncio')
```

Comment:
I think we should keep this mark based on https://github.com/pytest-dev/pytest-asyncio/issues/124 where folks are asking for pytest-asyncio to stop implicit marking itself, since it causes compatibility problems. There's no harm in having it, as far as I can tell.

Reply:
Ah okay!